src/client-services/github.client-service.test.tsx

```diff
@@ -0,0 +1,69 @@
+import axios, { AxiosInstance, AxiosResponse } from 'axios';
+import {
+  axiosConfig,
+  getReposList,
+  getRepoInfo,
+} from '../client-services/github.client-service';
+
+jest.mock('axios');
+
+describe('Github client service test', () => {
+  const accessToken = 'testToken';
+  const mockResponse: AxiosResponse = {
```

Comment:
I suggest changing it to the following.

`
  const mockResponse = {
    data: [{ name: 'openbc-web' }],
    headers: {
      Authorization: `token ${accessToken}`,
      'Content-Type': 'application/json',
    },
  };
`

Using the explicit type is causing type errors. Do you see it?

![image](https://github.com/OpenBCca/openbc-web/assets/40151580/6c5692f3-e6a4-41c3-823c-ba905034285e)

Reply:
Yes, i had the errors too, will remove the type